src/__tests__/date.test.ts

```diff
@@ -32,3 +32,11 @@ test("min max getters", () => {
     beforeBenchmarkDate
   );
 });
+
+test("UTC parser", () => {
+  const schemaUTCDate = z.date().utc();
+
+  const date = new Date("2023-04-04");
+
+  expect(schemaUTCDate.parse(date).toString()).toEqual('Mon Apr 04 2023 00:00:00 GMT+0000 (Coordinated Universal Time)');
```

Comment:
I think this spec is decent for `toString`, but perhaps we can go a little further by testing the resulting `Date` object itself

Reply:
Yeah, I think that we can try another `Date` object scenarios. Like:
`new Date(2023, 4, 3)`
`new Date('2023/04/04')`
`new Date('04/04/2023')`